m4/ul.m4

```diff
@@ -590,3 +590,33 @@ AC_DEFUN([UL_TINFO_CHECK], [
        TINFO_CFLAGS=""])
   ])
 ])
+
+AC_DEFUN([_UL_SIZEOF_TIME_T], [
+  AC_COMPUTE_INT($1, [sizeof(time_t)], [[
+    #include <time.h>
+    #include <stdio.h>
+  ]])
+])
+
+dnl
+dnl UL_SYS_TIME64
+dnl
+dnl Ensure 64bit times are used.
+dnl
+AC_DEFUN([UL_SYS_TIME64], [
```

Comment:
FWIW, the next autoconf release will have macros for this.

Reply:
Nice, I'll switch to those.

Reply:
@thesamesam 
Is there an official/blessed way to backport macros from autoconf proper to a projects `m4/` directory?